cypress/e2e/awx/administration/instances.cy.ts

```diff
@@ -173,12 +173,12 @@ describe('Instances: Delete', () => {
   it('can remove an instance from instance list toolbar', () => {
     cy.intercept('PATCH', '/api/v2/instances/*').as('removedInstance');
     cy.get('[data-cy="actions-dropdown"]').click();
-    cy.get('[data-cy="remove-instance"]').should('have.attr', 'aria-disabled', 'true');
+    cy.get('[data-cy="remove-instance"] button').should('have.attr', 'aria-disabled', 'true');
```

Comment:
Why is button added here?
It may be needed, but I worry that it will make it harder on devs to figure out when they need something like this.
Any way to make it so that this is not needed?